### ISSUE TITLE
ModulNotFoundError

### ISSUE THREAD

**Issue by 1Bl4ck1** [RileyXX/IMDB-Trakt-Syncer#79]
### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Bug Description

Traceback (most recent call last):
  File "C:\Users\Narki\AppData\Local\Programs\Python\Python312\Lib\site-packages\IMDBTraktSyncer\IMDBTraktSyncer.py", line 15, in <module>
    from IMDBTraktSyncer import checkChromedriver
  File "C:\Users\Narki\AppData\Local\Programs\Python\Python312\Lib\site-packages\IMDBTraktSyncer\checkChromedriver.py", line 3, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Narki\AppData\Local\Programs\Python\Python312\Scripts\IMDBTraktSyncer.exe\__main__.py", line 4, in <module>
  File "C:\Users\Narki\AppData\Local\Programs\Python\Python312\Lib\site-packages\IMDBTraktSyncer\IMDBTraktSyncer.py", line 22, in <module>
    import checkChromedriver
ModuleNotFoundError: No module named 'checkChromedriver'

### Environment

Windows 11

I entered all my credentials to verifyCredentials 

### Screenshots

![cmd_xwWT5b5c5F](https://github.com/RileyXX/IMDB-Trakt-Syncer/assets/152782113/49d90905-58ed-429a-a315-955119807e2b)
![WindowsTerminal_ZeQ1kXCarR](https://github.com/RileyXX/IMDB-Trakt-Syncer/assets/152782113/f4f04852-bfbc-4da1-a2a0-9a6430effab9)

**Comment by RileyXX** [comment]
Hi thanks for reporting this. Looks like there is a required package that isn't being installed during setup. I see what the issue is.

To install the missing package run the following in command prompt:
```
python -m pip install setuptools
```

After doing that run the script again and see if that resolves it. I will implement a fix for this shortly.

**Comment by RileyXX** [comment]
This should be fixed now in [v1.8.6](https://github.com/RileyXX/IMDB-Trakt-Syncer/releases/tag/v1.8.6).

You can update using the following:
````
python -m pip install IMDBTraktSyncer
````

Let me know if that fixes it for you. If you are still having issues let me know and I'll take another look.

**Comment by RileyXX** [comment]
Glad to hear the original error is fixed. 

As for the IMDB captcha issue, the only solution is to login/logout on IMDB from your browser on the same device. If you run the script too many times in a short period then the captcha can show up more frequently. This is usually only temporary. If you tried logging in/out several times already on your browser and still get that error then the only other thing you can do is wait a couple hours or try again the next day. Usually this issue will resolve on its own.
